### PR TITLE
Adding details for logging in at  Management group scope

### PR DESCRIPTION
**Pull request by raiyanalam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,9 +181,43 @@ With the Azure login Action, you can perform an Azure login using [Azure service
   }
   
 ```
+  *  Alternatively,  to create service principal that has access over management group scope, run the below Azure CLI command.
```
Comment (Ganeshrockz) [inline README.md:184]: The  `sp` API with the `sdk-auth` flag will fail even if one of the scopes present is a management group. Should we rephrase this line to express that?

File: README.md [hunk 2]
```diff
@@ -112,6 +112,40 @@ jobs:
...
+      uses: actions/checkout@v2 
+
+    - name: Create or Update Azure Policies
+      uses: azure/manage-azure-policy@v0
+      with:      
+        paths:  |                
+          policies/**
+        
+```
+For deploying policies or initiatives at a management group level, the azure login action should have input `allow-no-subscriptions` set to true.
```
Comment (Ganeshrockz) [inline README.md:144]: definitons or initiatives?  Initiatives will fall under policies right?

File: README.md [hunk 3]
```diff
@@ -147,9 +181,43 @@ With the Azure login Action, you can perform an Azure login using [Azure service
...
+
+  {
+    "appId": "<GUID>",
+    "displayName": "<display-name>",
+    "name": "<url>",
+    "password": "<GUID>",
+    "tenant": "<GUID>"
+  }
+  
+  # copy the GUID values for appId, password and tentant from above JSON and replace them in the following JSON. Once replaced, copy the JSON to clipboard
```
Comment (Ganeshrockz) [inline README.md:204]: Typo for tenant

**Review by Ganeshrockz — APPROVED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (raiyanalam) [reply README.md]: updated. Thanks

File: README.md (hunk 3 above)
Comment (raiyanalam) [reply README.md]: Good catch. Updated it.

File: README.md (hunk 2 above)
Comment (raiyanalam) [reply README.md:144]: Not really. Here we are following the hierarchy in policy UI. Eg, Initiative definitions or Policy definitions

**Pull request merged**